community/communities.md

```diff
@@ -89,3 +89,21 @@ Link: [https://stackoverflow.com/questions/tagged/machine-translation](https://s
 The **Machine-Translation** tag on Quora gathers user questions and answers related to machine translation, including its development and performance.
 
 Link: [https://www.quora.com/topic/Machine-Translation/](https://www.quora.com/topic/Machine-Translation/)
```

Comment:
I'd remove *Link:* here and throughout, also remove the *https://www.* from the visible part.